src/sgame/sg_main.cpp

```diff
@@ -2991,9 +2991,20 @@ void G_RunFrame( int levelTime )
 void G_PrepareEntityNetCode() {
```

Comment:
It's not so much that the information is out of date, but rather that the information can be completely wrong. HandlePrepareNetcode can do anything. Currently, it mainly just clamps the float health to an int, but that may not be what happens in the future. That's why we must call call PrepareNetcode after all the state has been written.